leakpro/attacks/mia_attacks/lira.py

```diff
@@ -112,36 +147,20 @@ def prepare_attack(self:Self)->None:
             count_in_samples = np.count_nonzero(self.in_indices_mask)
             if count_in_samples > 0:
                 self.logger.info(f"Some shadow model(s) contains {count_in_samples} IN samples in total for the model(s)")
-                self.logger.info("This is not an offline attack!")
+                self.logger.info("This is not an true offline attack!")
```

Comment:
?

Reply:
Had a bug (when I did the offline attack) where some shadow models had IN-samples. Also a good sanity check so users dont mix up offline attacks with online models.